extensions/eclipse-che-theia-workspace/src/node/che-workspace-server.ts

```diff
@@ -34,7 +34,7 @@ export class CheWorkspaceServer extends DefaultWorkspaceServer {
   // if not possible, use default method
   protected async getRoot(): Promise<string | undefined> {
     const workspace = await this.workspaceService.currentWorkspace();
-    if (!isMultiRoot(workspace)) {
+    if (workspace.devfile?.attributes?.multiRoot === 'off') {
```

Comment:
I notice that in `che-workspace-server` we're using `devfile.attributes` hile in the plug-in we're using `devfile.metadata.attributes`

Reply:
good catch!
Looks like, first [it was consistent](https://github.com/eclipse/che-theia/pull/778/files#diff-0fac5b1393746869f861ca2c028c6464843b435b3b5643f675c5f79a3440c7caR174)
and then, I guess, it was changed within another PR.

Reply:
I added couple logs to check that behavior and found that fro `class WorkspaceProjectsManager`:
- the attribute `multiRoot` comes really from `devfile.metadata?.attributes`
- the attribute doesn't come within `devfile.attributes`

![metadata](https://user-images.githubusercontent.com/5676062/112628442-94776780-8e3b-11eb-89d6-fd6b907c4e15.png)

Reply:
My logs:
![logs](https://user-images.githubusercontent.com/5676062/112628640-d3a5b880-8e3b-11eb-8660-2101b86ca52a.png)

Reply:
yes for now devfileV2 has not global attributes :-/
so it has been asked to the devfile spec to add it back

https://github.com/devfile/api/pull/352#issuecomment-806857030

so in short: for now, look from metadata.attributes
in the future, we'll be able to look it from devfile.attributes

Reply:
so, atm I can merge my PR as is - correct?

Reply:
yes correct
But we should still use DevfileService instead of WorkspaceService as `workspace.devfile` is not there in DevWorkspace engine

Reply:
so, we need to replace `WorkspaceService ` by `DevfileService` [here](https://github.com/eclipse/che-theia/blob/6429b29476bf7825189aac98343e68bb3e06a2df/extensions/eclipse-che-theia-workspace/src/node/che-workspace-server.ts#L31) to have it working for DevWorkspace engine, right? 
if so - I can prepare a PR for that.

Reply:
yes correct and then use .metadata.attributes

it'll work then as expected with DevWorkspace engine